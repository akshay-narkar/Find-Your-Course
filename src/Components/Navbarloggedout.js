import { Link } from 'react-router-dom';

export default function Navloggedout() {
  return (
    <>
      <li>
        <Link to={{ pathname: '/login' }} className="nav-link active fw-light" aria-current="page">Login</Link>
      </li>
      <li className="nav-item">
        <Link to={{ pathname: '/signup' }} className="nav-link active fw-light" aria-current="page">SignUp</Link>
      </li>

    </>

  );
}
