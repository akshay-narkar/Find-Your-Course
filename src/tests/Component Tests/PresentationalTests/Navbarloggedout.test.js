/*eslint-disable*/
import renderer from 'react-test-renderer';
import { BrowserRouter } from 'react-router-dom';
import Navbarloggedout from '../../../Components/Navbarloggedout';

it('UI is unaltered wrt first snapshot', () => {
  const elem = renderer.create(<BrowserRouter><Navbarloggedout /></BrowserRouter>).toJSON();

  expect(elem).toMatchSnapshot();
});
