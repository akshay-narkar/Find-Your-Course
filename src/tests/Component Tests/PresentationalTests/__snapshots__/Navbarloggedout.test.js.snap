// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UI is unaltered wrt first snapshot 1`] = `
Array [
  <li>
    <a
      aria-current="page"
      className="nav-link active fw-light"
      href="/login"
      onClick={[Function]}
    >
      Login
    </a>
  </li>,
  <li
    className="nav-item"
  >
    <a
      aria-current="page"
      className="nav-link active fw-light"
      href="/signup"
      onClick={[Function]}
    >
      SignUp
    </a>
  </li>,
]
`;
